src/client/components/contact-page/Contact.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import image from '../../assets/images/image26.png';
+import './contact.css';
+
+export default function ContactPage() {
```

Comment:
It is preferrable to create components that are only concerned with one thing. Its _probably_ not going to be critical here, but once the code base starts to grow, we need to start reusing components, make changes and move things around.

It is demonstrated here:
https://reactjs.org/docs/thinking-in-react.html

Reply:
Yes, I understand and I note that.